plugins/arDominionB5Plugin/modules/browse/templates/hierarchySuccess.php

```diff
@@ -1,19 +1,20 @@
 <?php decorate_with('layout_1col.php'); ?>
 <?php slot('title'); ?>
 
-  <div class="d-flex justify-content-end flex-wrap gap-2 ms-auto">
+  <h1>
+    <?php echo __('Hierarchy'); ?>    
+  </h1>
+
+  <div class="d-flex flex-wrap gap-2 mb-3">
     <input type="button" id="fullwidth-treeview-reset-button" class="btn btn-sm atom-btn-white" value="<?php echo __('Reset'); ?>" />
-    <input type="button" id="fullwidth-treeview-more-button" class="btn btn-sm atom-btn-white" data-label="<?php echo __('%1% more'); ?>" value="" />
+    <input type="hidden" id="fullwidth-treeview-more-button" class="btn btn-sm atom-btn-white" data-label="<?php echo __('%1% more'); ?>" value="" />
   </div>
 
-  <?php echo image_tag('/vendor/jstree/themes/default/throbber.gif', ['id' => 'fullwidth-treeview-activity-indicator', 'alt' => __('Loading ...')]); ?>
```

Comment:
Are you adding the spinner differently? Isn't this used from the JS?

Reply:
I have updated this so that during page load it uses the fa throbber by the section title. Still uses the jstree throbber within the treeview itself when building the treeview nodes.